design/iterations.go

```diff
@@ -24,6 +24,9 @@ var iterationAttributes = a.Type("IterationAttributes", func() {
 	a.Attribute("name", d.String, "The iteration name", func() {
 		a.Example("Sprint #24")
 	})
+	a.Attribute("description", d.String, "Description of the iteration ", func() {
+		a.Example("this is the description of iteration")
```

Comment:
Dammm ! That was so simple, I should have done that already.
Now, fixed in https://github.com/almighty/almighty-core/pull/652/commits/6a004852b083521ec2d390e83dccfadc911969f0